ci-operator/step-registry/ipi/conf/openstack/ipi-conf-openstack-ref.yaml

```
@@ -6,15 +6,16 @@ ref:
     requests:
       cpu: 10m
       memory: 100Mi
+  credentials:
+  - namespace: test-credentials
+    name: cluster-secrets-openstack-vexxhost
+    mount_path: /output/.config/openstack/vexxhost
```

The cluster type for vexxhost is `openstack-vexxhost`.
```suggestion
    mount_path: /output/.config/openstack/openstack-vexxhost
```